newIDE/app/src/UI/CustomSvgIcons/CircledInfo.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
you seem to not use this file?

Reply:
There is a memo on the icons.

Reply:
I think he meant the file "CircledInfo.js" itself

Reply:
Oh, ok, yes, but it's the Icon with the right margins. I guess it could be confusing not to have it. I'm sure it will be useful at some point.